Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM eclipse-temurin:17_35-jdk-focal
```

Comment:
Can you add docker.io/ to make it a full container name. This makes it easier for e.g. podman on macOS which runs without a tty to ask for a location of the repo.

Reply:
added